awx/main/tasks.py

```diff
@@ -1600,6 +1613,8 @@ def should_use_proot(self, job):
         '''
         Return whether this task should use proot.
         '''
+        if job.is_containerized:
+            return False # TODO: Get smarter
```

Comment:
Is this `TODO` still needed?  It seems reasonable to me that containerized jobs would just skip bwrap entirely.